src/pages/home/report/ReportActionCompose.js

```diff
@@ -353,13 +354,25 @@ class ReportActionCompose extends React.Component {
         }
     }
 
+    /**
+     * Whether virtual keyboard is open or not
+     * @returns {Boolean}
+     */
+    isVirtualKeyboardOpen() {
```

Comment:
Can you please move this to it's own `VirtualKeyboard` library to make it more discoverable?

Reply:
You should have it return `true` or `false` if the VirtualKeyboard API is available, or null if it's not. Do not fall back on `canUseTouchScreen` within the library.

Like this:
#### src/libs/virtualKeyboard.js
```js
/**
 * Is the virtual keyboard open?
 *
 * @returns {Boolean|null} – null if the VirtualKeyboard API is unavailable
 */
function isOpen() {
    if (!_.has(navigator, 'virtualKeyboard')) {
        return null;
    }
    return navigator.virtualKeyboard.boundingRect.y > 0;
}

export {
    isOpen,
};
```

#### src/pages/home/report/ReportActionCompose.js
```js
...
...
import * as VirtualKeyboard from '../../../libs/virtualKeyboard';
...
...

/**
 * @returns {Boolean}
 */
shouldAssumeVirtualKeyboardIsOpen() {
    const isOpen = VirtualKeyboard.isOpen();
    return _.isNull(isOpen) ? canUseTouchScreen() : isOpen;
}

/**
 * Listens for keyboard shortcuts and applies the action
 *
 * @param {Object} e
 */
triggerHotkeyActions(e) {
    if (!e || this.shouldAssumeVirtualKeyboardIsOpen()) {
        return;
    }
    ...
    ...
}
```

Reply:
+1.

Reply:
ok done